tests/sys/psa_crypto_aead/example_aead_aes_128.c

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (C) 2025 TU Dresden
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     tests
+ * @{
+ *
+ * @brief       Tests the PSA aead configurations
+ *
+ * @author      Lukas Luger <lukas.luger@mailbox.tu-dresden.de>
+ *
+ * @}
+ */
+
+#include <stdio.h>
+#include <stdint.h>
+#include "od.h"
+#include "psa/crypto.h"
+
+#define AES_128_KEY_SIZE    (16)
+
+static const uint8_t KEY_128[] = {
+    0x40, 0x41, 0x42, 0x43, 0x44, 0x45, 0x46, 0x47,
+    0x48, 0x49, 0x4a, 0x4b, 0x4c, 0x4d, 0x4e, 0x4f
+};
+/* certain PSA backends require the data to be in RAM rather than ROM
+ * so these values cannot be `const` */
+static uint8_t NONCE[] = {
+    0x10, 0x11, 0x12, 0x13, 0x14, 0x15, 0x16
+};
+static uint8_t NONCE_LEN = 7;
+
+static uint8_t ADDITIONAL_DATA[] = {
+    0x00, 0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07
+};
+static uint8_t ADDITIONAL_DATA_LEN = 8;
+
+static uint8_t PLAINTEXT[] = {
+    0x20, 0x21, 0x22, 0x23
+};
+static uint8_t PLAINTEXT_LEN = 4;
+
+static uint8_t CIPHERTEXT[] = {
+    0x71, 0x62, 0x01, 0x5b, 0x4d, 0xac, 0x25, 0x5d
+};
+static uint8_t CIPHERTEXT_LEN = 8;
+/**
+ * @brief   Example function to perform an AES-128 CCM encryption and decryption
+ *          with the PSA Crypto API.
+ *
+ * @return  psa_status_t
+ */
+psa_status_t example_aead_aes_128_ccm(void)
+{
+    psa_status_t status = PSA_ERROR_DOES_NOT_EXIST;
+    psa_key_id_t key_id = 0;
+    psa_key_attributes_t attr = psa_key_attributes_init();
+    psa_key_usage_t usage = PSA_KEY_USAGE_ENCRYPT | PSA_KEY_USAGE_DECRYPT;
+    psa_algorithm_t algo = PSA_ALG_AEAD_WITH_SHORTENED_TAG(PSA_ALG_CCM, 4);
+    size_t encr_output_size = PSA_AEAD_ENCRYPT_OUTPUT_SIZE(PSA_KEY_TYPE_AES,
```

Comment:
```suggestion
    const size_t encr_output_size = PSA_AEAD_ENCRYPT_OUTPUT_SIZE(PSA_KEY_TYPE_AES,
```

if possible, to avoid VLAs